### PR TITLE
RCPS-7 web-frontend-maintenance: Refactor Login Module

### PR DESCRIPTION
**Pull request by HaroldDinglasan**

### CODE REVIEW

File: FrontEnd/login-page/src/logo.svg
Comment (esdi-00) [inline FrontEnd/login-page/src/logo.svg:1]: why is the react logo in the project? is this necessary?

File: FrontEnd/login-page/.vscode/settings.json
Comment (esdi-00) [inline FrontEnd/login-page/.vscode/settings.json:1]: why is there vscode specific files in the project?

**Review by esdi-00 — CHANGES_REQUESTED** [review]
see comments

File: FrontEnd/login-page/.vscode/settings.json
Comment (HaroldDinglasan) [reply FrontEnd/login-page/.vscode/settings.json:1]: as far as i know po sir it is already built in

**Pull request closed without merging**